packages/web/src/pages/sign-on/components/desktop/MobileContentContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { HTMLAttributes, PropsWithChildren } from 'react'
+
+import { Flex, FlexProps } from '@audius/harmony'
+import cn from 'classnames'
+
+import styles from './MobileContentContainer.module.css'
+
+export type MobileContentContainerProps = PropsWithChildren<FlexProps> &
+  HTMLAttributes<HTMLDivElement>
+
+export const MobileContentContainer = ({
```

Comment:
container for this
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/523ff9ce-69b8-4595-863b-e812108fcd30)